Sources/ArcGISToolkit/Components/FeatureFormView/AttachmentHelpers/AttachmentPhotoPicker.swift

```diff
@@ -32,7 +32,10 @@ struct AttachmentPhotoPicker: ViewModifier {
             .photosPicker(
                 isPresented: $photoPickerIsPresented,
                 selection: $item,
-                matching: .any(of: [.images, .not(.livePhotos)])
+                matching: .any(of: [
+                    .images,
+                    .videos
+                ])
```

Comment:
Do we still want to disallow live photos?

Reply:
Based on your description I assume this was intended.

Reply:
Yes, intended. I tried in on-device and it worked. I think it simply took the "key frame" and used that.  Most of the image/video types I tried worked successfully, with the exception of videos that come in with the `.quicktime` extension, which don't get applied to the database, but do show up in the in-memory list of attachments.

Reply:
There will be another PR listing the issues I've encountered.